packages/docs/docs/usage/widgets.md

```diff
@@ -78,16 +78,16 @@ The built-in string field also supports the JSON Schema `format` property, and w
 - `data-url`: By default, an `input[type=file]` element is used; in case the string is part of an array, multiple files will be handled automatically (see [File widgets](#file-widgets)).
 - `date`: By default, an `input[type=date]` element is used;
 - `date-time`: By default, an `input[type=datetime-local]` element is used.
+- `time`: By default an `input[type=time]` element is used;
 
 ![](https://i.imgur.com/xqu6Lcp.png)
 
-Please note that, even though they are standardized, `datetime-local` and `date` input elements are not yet supported by Firefox and IE. If you plan on targeting these platforms, two alternative widgets are available:
+Please note that, even though they are standardized, `datetime-local`, `date` and `time` input elements are not yet supported by IE.
+If you plan on targeting this platforms, two alternative widgets are available:
```

Comment:
I don't think IE is ever going to get that support 🙂. I'm tempted to just remove the IE-specific blurb entirely.

```suggestion
Please note that, even though they are standardized, `datetime-local`, `date` and `time` input elements are not supported by IE. If you plan on targeting IE, two alternative widgets are available:
```